package.json

```diff
@@ -17,7 +17,7 @@
   },
   "scripts": {
     "serve": "npm run mockserver && polyserve",
-    "polyserve": "polyserve -o --proxy-target http://127.0.0.1:9999/sc --proxy-path sc",
+    "polyserve": "polyserve --proxy-target http://127.0.0.1:9999/sc --proxy-path sc --hostname 0.0.0.0",
```

Comment:
hmmmm, I'm afraid this isn't possible, `0.0.0.0` isn't a routable IP address, it's an equivilent of [IPAddress.Any](https://msdn.microsoft.com/en-us/library/system.net.ipaddress.any(v=vs.110).aspx) in .NET. It allows me to access tests from another machine. It's only usable to accept connections (inside `Socket.bind()`), and can't be used to initiate connections (inside `Sockent.connect()`). So we need both 0.0.0.0 and 127.0.0.1, since the tests runner's IP isn't determinable.

Reply:
Now I understand, thanks!